Src/BootCamp.Chapter/BalanceStats.cs

```diff
@@ -7,31 +12,198 @@ public static class BalanceStats
         /// </summary>
         public static string FindHighestBalanceEver(string[] peopleAndBalances)
         {
-            return "";
+            if (peopleAndBalances == null || peopleAndBalances.Length == 0) return "N/A.";
+
+            int highestBalance = 0;
+            List<String> listOfPeople = new List<String>();
```

Comment:
This is going to be a bit pedantic: `listOfPeople` is not a great variable name. We know it's a list, it's right there on the left and right of it. `people` would work just fine. There's no need to emphasize mechanism over intent.